homeassistant/components/landisgyr_heat_meter/const.py

```diff
@@ -1,212 +1,16 @@
-    ),
-    SensorEntityDescription(
-        key="flow_temperature_max_c",
-        name="Flow temperature max",
-        native_unit_of_measurement=UnitOfTemperature.CELSIUS,
-        device_class=SensorDeviceClass.TEMPERATURE,
-        entity_category=EntityCategory.DIAGNOSTIC,
-    ),
-    SensorEntityDescription(
-        key="flow_temperature_max_previous_year_c",
-        name="Flow temperature max previous year",
-        native_unit_of_measurement=UnitOfTemperature.CELSIUS,
-        device_class=SensorDeviceClass.TEMPERATURE,
-        entity_category=EntityCategory.DIAGNOSTIC,
-    ),
-    SensorEntityDescription(
-        key="operating_hours",
-        name="Operating hours",
-        device_class=SensorDeviceClass.DURATION,
-        native_unit_of_measurement=UnitOfTime.HOURS,
-        entity_category=EntityCategory.DIAGNOSTIC,
-    ),
-    SensorEntityDescription(
-        key="flow_hours",
-        name="Flow hours",
-        device_class=SensorDeviceClass.DURATION,
-        native_unit_of_measurement=UnitOfTime.HOURS,
-        entity_category=EntityCategory.DIAGNOSTIC,
-    ),
-    SensorEntityDescription(
-        key="fault_hours",
-        name="Fault hours",
-        device_class=SensorDeviceClass.DURATION,
-        native_unit_of_measurement=UnitOfTime.HOURS,
-        entity_category=EntityCategory.DIAGNOSTIC,
-    ),
-    SensorEntityDescription(
-        key="fault_hours_previous_year",
-        name="Fault hours previous year",
-        device_class=SensorDeviceClass.DURATION,
-        native_unit_of_measurement=UnitOfTime.HOURS,
-        entity_category=EntityCategory.DIAGNOSTIC,
-    ),
-    SensorEntityDescription(
-        key="yearly_set_day",
-        name="Yearly set day",
-        icon="mdi:clock-outline",
-        entity_category=EntityCategory.DIAGNOSTIC,
-    ),
-    SensorEntityDescription(
-        key="monthly_set_day",
-        name="Monthly set day",
-        icon="mdi:clock-outline",
-        entity_category=EntityCategory.DIAGNOSTIC,
-    ),
-    SensorEntityDescription(
-        key="meter_date_time",
-        name="Meter date time",
-        icon="mdi:clock-outline",
-        device_class=SensorDeviceClass.TIMESTAMP,
-        entity_category=EntityCategory.DIAGNOSTIC,
-    ),
-    SensorEntityDescription(
-        key="measuring_range_m3ph",
-        name="Measuring range",
-        native_unit_of_measurement=UnitOfVolumeFlowRate.CUBIC_METERS_PER_HOUR,
-        icon="mdi:water-outline",
-        entity_category=EntityCategory.DIAGNOSTIC,
-    ),
-    SensorEntityDescription(
-        key="settings_and_firmware",
-        name="Settings and firmware",
-        entity_category=EntityCategory.DIAGNOSTIC,
-    ),
-)
+# Keys used to check for GJ or MWH
+MWH_IDENTITY_KEY = "heat_usage_mwh"
+GJ_IDENTITY_KEY = "heat_usage_gj"
+
+# Keys that are energy unit specific
+MWH_ONLY_KEYS = ["heat_usage", "heat_previous_year"]
+GJ_ONLY_KEYS = ["heat_usage_gj", "heat_previous_year_gj"]
```

Comment:
Slight stylistic nitpick: I would use a set here since you're using this to test for containment and not as something to iterate over (in order).

```suggestion
MWH_ONLY_KEYS = {"heat_usage", "heat_previous_year"}
GJ_ONLY_KEYS = {"heat_usage_gj", "heat_previous_year_gj"}
```

Reply:
Thanks! Commited!